<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainMenu</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>8</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="MENU">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="Standard">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
             </property>
             <property name="text">
              <string>PRZEGLĄDAJ OFERTY</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="LM">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
             </property>
             <property name="text">
              <string>OFERTY LAST MINUTE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="LogRej">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
             </property>
             <property name="text">
              <string>LOGOWANIE/REJESTRACJA</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="MojeOf">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
             </property>
             <property name="text">
              <string>MOJE OFERTY</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="OptionsBut">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
             </property>
             <property name="text">
              <string>OPCJE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Exit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
             </property>
             <property name="text">
              <string>WYJDŹ</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="TytulLog">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>221</width>
          <height>81</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Zaloguj się do BiuroPodrozy</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="NazwaLogWpisz">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="HasloLogWpisz">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
        </property>
       </widget>
       <widget class="QLabel" name="TytulRej">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>210</y>
          <width>221</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Zarejestruj sie </string>
        </property>
       </widget>
       <widget class="QLabel" name="NazwaLog">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>133</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Nick</string>
        </property>
       </widget>
       <widget class="QLabel" name="HasloLog">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>110</y>
          <width>133</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Hasło</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Zaloguj">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
        </property>
        <property name="text">
         <string>Zaloguj</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Zaloguj_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>500</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
        </property>
        <property name="text">
         <string>Zarejestruj</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>300</y>
          <width>135</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="ImieRej">
           <property name="text">
            <string>Imię</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ImieWpiszRej">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>300</y>
          <width>135</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="NazwRej">
           <property name="text">
            <string>Nazwisko</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NazwiskoWpiszRej">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>371</y>
          <width>135</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="HasloRej">
           <property name="text">
            <string>Hasło</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="HasloWpiszRej">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>440</y>
          <width>135</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="PINRej">
           <property name="text">
            <string>PIN karty</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PINKartyWpiszRej">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>371</y>
          <width>135</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="NazwaRej">
           <property name="text">
            <string>Nick</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NazwaWpiszRej">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>440</y>
          <width>135</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="NrKartyRej">
           <property name="text">
            <string>Nr karty kredytowej</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NrKartyKredWpiszRej">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="TytulLog_2">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>50</y>
          <width>281</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Podaj kod aplikacji, aby zostać administratorem</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="KodWpisz">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
        </property>
       </widget>
       <widget class="QLabel" name="HasloLog_2">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>110</y>
          <width>133</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Kod</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Zaloguj_3">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>170</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="Oferty">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>471</width>
          <height>91</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ravie</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Moje Podróze:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ReturnfromOffers">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>500</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
        </property>
        <property name="text">
         <string>POWRÓT</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="ScrollOffers">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>110</y>
          <width>971</width>
          <height>371</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>952</width>
           <height>369</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10"/>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="trick">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="Checker">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="littleChecker_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="littleChecker_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>800</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
